pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -50,10 +50,28 @@ msgctxt "#30007"
 msgid "Response timeout in seconds"
 msgstr ""
 
-#empty strings from id 30008 to 30099
+msgctxt "#30008"
+msgid "Advanced settings"
+msgstr ""
+
+#empty strings from id 30009 to 30049
+
+msgctxt "#30050"
+msgid "Auto recordings"
+msgstr ""
+
+msgctxt "#30051"
+msgid "Use approximate start time instead of fixed start time"
```

Comment:
This does not hit the point. Sounds, like the current approach would only match broadcasts starting at exactly at the given start time, which is not the case. I'm still squeezing my head to get something (imo) better out of it... ;-)